core/field_image.js

```diff
@@ -59,14 +59,16 @@ Blockly.FieldImage = function(src, width, height,
   }
 
   // Ensure height and width are numbers.  Strings are bad at math.
-  this.height_ = Number(height);
-  this.width_ = Number(width);
-  if (this.height_ <= 0 || this.width_ <= 0) {
+  var imageHeight = Number(height);
+  var imageWidth = Number(width);
+  if (imageHeight <= 0 || imageWidth <= 0) {
     throw Error('Height and width values of an image field must be greater' +
       ' than 0.');
   }
-  this.size_ = new Blockly.utils.Size(this.width_,
-      this.height_ + 2 * Blockly.BlockSvg.INLINE_PADDING_Y);
+  // Store the image height, since it is different from the field height.
+  this.imageHeight_ = imageHeight;
+  this.size_ = new Blockly.utils.Size(imageWidth,
+      imageHeight + Blockly.FieldImage.Y_PADDING);
```

Comment:
Does this mean there's only 1/2 a pixel above and below an image or should this be 2*Y_PADDING?

Reply:
Only below the image.